packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -293,40 +393,130 @@ export const Webchat = forwardRef((props, ref) => {
   }, [webchatState.isWebchatOpen])
 
   const textArea = useRef()
-  const staticAssetsUrl = getScriptBaseURL()
 
-  const CustomTriggerButton = webchatState.theme.customTriggerButton
-  let logoUrl = Logo
-  if (props.theme && props.theme.brandIconUrl)
-    logoUrl = props.theme.brandIconUrl
-  if (webchatState.theme && webchatState.theme.brandIconUrl)
-    logoUrl = webchatState.theme.brandIconUrl
+  const CustomTriggerButton = webchatState.theme.customTrigger
+  let triggerImage = Logo
+  if (props.theme && 'triggerButtonImage' in props.theme)
+    triggerImage = props.theme.triggerButtonImage
+  if (webchatState.theme && 'triggerButtonImage' in webchatState.theme)
+    triggerImage = webchatState.theme.triggerButtonImage
   const triggerButton = CustomTriggerButton ? (
     <CustomTriggerButton />
   ) : (
+    <StyledTriggerButton
+      style={{
+        ...webchatState.theme.triggerButtonStyle
+      }}
+    >
+      {triggerImage && (
+        <img
+          style={{
+            height: 50
```

Comment:
maybe put some option here in the `style` props for settings heigth & width. Common used behaviour.